training/scaleway_gpu.py

```
@@ -0,0 +1,600 @@
+					"returncode": result.returncode,
+					"success": result.returncode == 0
+				}
+		except subprocess.TimeoutExpired as e:
+			return {
+				"stdout": e.stdout.decode() if e.stdout else "",
+				"stderr": e.stderr.decode() if e.stderr else "",
+				"returncode": -1,
+				"success": False,
+				"error": f"Command timed out after {timeout} seconds"
+			}
+		except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)